source/install/install-rhel-66-mysql.rst

```diff
@@ -0,0 +1,66 @@
+..  _install-rhel-66-mysql:
+
+Installing MySQL Database Server
+================================
+
+Install and set up the database for use by the Mattermost server. You can install either MySQL or PostgreSQL. To install PostgreSQL, see :ref:`install-rhel-66-postgresql`.
```

Comment:
This is the same for all the other links referencing subsections on the same guide. Not sure if you'd have any thoughts on that?